.github/workflows/linux.yml

```diff
@@ -1009,6 +1061,7 @@ jobs:
           TEST_PRECISION: FP16
 
       - name: Python Frontend tests
+        if: fromJSON(needs.smart_ci.outputs.affected_components).PyTorch_FE.test
```

Comment:
we also have Paddle FE tests there https://github.com/openvinotoolkit/openvino/tree/master/tests/layer_tests/py_frontend_tests

Reply:
Added running this step also for PDPD_FE